### DISCUSSION TITLE
Redis

### DISCUSSION THREAD

**Discussion by agarwalrohit2503** [serverpod/serverpod#291] [Q&A] [answered]
Is there any option to just launch the docker instance only using Postgres and not with Redis DB

**Comment by vlidholt** [comment, marked as answer]
The easiest way is to just edit it out of the `docker-compose.yaml` file.

**Reply by agarwalrohit2503** [reply]
Even if i commented the part of redis iin docker-compose.yaml
It is still waiting for redis db
![Screenshot 2022-09-14 at 3 09 18 PM](https://user-images.githubusercontent.com/33911927/190119839-5852890c-5f27-44b6-b60f-a6376f7ac0d1.png)
![Screenshot 2022-09-14 at 3 11 06 PM](https://user-images.githubusercontent.com/33911927/190119998-66db76b7-8d37-43bd-af26-496b326a06b7.png)
